app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -173,12 +173,50 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_defaultRecentlyPlayedToolbarTitleIsDisplayed() {
```

Comment:
This test was incorrectly passing on robolectic and was failing on espresso.

Title on RecentlyPlayedActivtiy changes depending upon the chapters played. But it takes time to update the title. Adding delay after launching the activity makes this test fail on robolectric too. 

This test is now passing.